tests/formik/EmailInput.test.js

```diff
@@ -87,7 +87,7 @@ describe("formik/Input", () => {
     const onSubmit = jest.fn();
     render(<TestEmailInputForm onSubmit={onSubmit} />);
     const emailInput = screen.getByRole("combobox");
-    userEvent.paste(emailInput, "john.doe@email.com  sam.doeemail.com");
+    userEvent.paste(emailInput, "sam.doeemail.com");
```

Comment:
@mohit-s96 is this change required?

Reply:
Hi,
The new format requires us to accept input like `John Doe <john@email.com>`, pick out `john@email.com` and discard the rest i.e. `John Doe <>`. Thus this input is considered valid (only if pasted through the clipboard). 

The issue here is that this test asserts that `john.doe@email.com  sam.doeemail.com` should be invalid when pasted from the clipboard, which wouldn't pass because there is at least one valid email present in the pasted input.

Another way of making it pass is using the `userEvent.type` method instead of `userEvent.paste`.